ouroboros-consensus/changelog.d/20230612_112138_armandoifsantos_dynamic_block_forging.md

```diff
@@ -0,0 +1,38 @@
+<!--
+A new scriv changelog fragment.
+
+Uncomment the section that is right (remove the HTML comment wrapper).
+-->
+
+<!--
+### Patch
+
+- A bullet item for the Patch category.
+
+-->
+### Non-Breaking
+
+- `addBlockRunner` now if interrupted, will notify all blocked threads.
+
+- Added `closeBlocksToAdd` function
+
+### Breaking
+
+- Removed the `pInfoBlockForging` record field from the `ProtocolInfo` type.
+  - Refactor code to accommodate the extraction of `pInfoBlockForging` record field from the
```

Comment:
I don't know if we need to mention that we refactor the code, maybe this should be obvious from the fact that a field from a data structure was removed.